ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/Destination.java

```diff
@@ -60,8 +70,12 @@ public boolean equals(Object obj) {
 				return false;
 		} else if (!reasonForTravel.equals(other.reasonForTravel))
 			return false;
+		if (geolocation == null) {
```

Comment:
All of our `.equals()` and `.hashCode()` implementations should use EqualsBuilder and HashCodeBuilder. See ExpenseClaim for an example.